services/kommander-ui/metadata.yaml

```
@@ -0,0 +1,5 @@
+type: internal
+scope:
+  - workspace
+dependencies:
```

```suggestion
requiredDependencies:
```

I think it needs to be `requiredDependencies` bc there is a HR dependency here which will always block deployment